cmd/minishift/cmd/start_preflight.go

```diff
@@ -37,23 +36,25 @@ import (
 	"github.com/minishift/minishift/pkg/util/os/atexit"
 	"github.com/spf13/viper"
 
-	cmdUtils "github.com/minishift/minishift/cmd/minishift/cmd/util"
+	cmdUtil "github.com/minishift/minishift/cmd/minishift/cmd/util"
 	minishiftNetwork "github.com/minishift/minishift/pkg/minishift/network"
 	openshiftVersion "github.com/minishift/minishift/pkg/minishift/openshift/version"
 	stringUtils "github.com/minishift/minishift/pkg/util/strings"
 )
 
 const (
 	StorageDisk   = "/mnt/?da1"
-	GithubAddress = "github.com:http"
+	GithubAddress = "https://github.com"
```

Comment:
It's important that we actually try to connect via `https` because that's also what we'll use for all the github.com communication later on. `http` is not enough!